src/ape_pm/compiler.py

```diff
@@ -5,11 +5,17 @@
 from ethpm_types import ABI, ContractType
 from pydantic import parse_obj_as
 
-from ape.api import CompilerAPI
+from ape.api import CompilerAPI, ConfigItem
 from ape.exceptions import CompilerError
 
 
+class InterfaceCompilerConfig(ConfigItem):
+    base_path: Path = Path("contracts")
```

Comment:
I like this addition, but it might need to be in core to look right w/ the config e.g.

```yaml
pm:
  base_path: contracts
```

versus maybe something more like this:

```yaml
project:
  contracts_folder: ./contracts
```

Reply:
Yeah that definitely looks better. However, we will need a nicer way to get at the `config_manager` in `CompilerAPI`. We have to do this same thing in `ape-solidity` and `ape-vyper`.

Also, is there ever a chance that a user will want to use a different base path for `ape-pm` versus `ape-solidity` / `ape-vyper`?